<launch>
    <arg name="timeout_sec" default="0.05"/>
    <arg name="point_cloud_topic" default="/luminar_pcl_cutted"/>
    <arg name="point_cloud_frame_id" default="luminar_lidar_0"/>
    <arg name="camera_topics"
         default="[/zed/zed_node/left/]"/>
    <arg name="image_type" default="1" description="0:RAW, 1:RECTIFIED"/>
    <arg name="use_compressed_image" default="true"/>
    <arg name="image_topic_last_name" default="image_rect_color/compressed"/>
    <arg name="camera_info_topic_last_name" default="camera_info"/>

    <node pkg="color_point_cloud" exec="color_point_cloud" name="color_point_cloud" output="screen">
        <param name="use_sim_time" value="true"/>
        <param name="timeout_sec" value="$(var timeout_sec)"/>
        <param name="point_cloud_topic" value="$(var point_cloud_topic)"/>
        <param name="point_cloud_frame_id" value="$(var point_cloud_frame_id)"/>
        <param name="camera_topics" value="$(var camera_topics)"/>
        <param name="image_type" value="$(var image_type)"/>
        <param name="use_compressed_image" value="$(var use_compressed_image)"/>
        <param name="image_topic_last_name" value="$(var image_topic_last_name)"/>
        <param name="camera_info_topic_last_name" value="$(var camera_info_topic_last_name)"/>
    </node>
</launch>
